keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -469,45 +453,42 @@ type LeaseSpec struct {
 For the identity leases, a new lease will be created
 
 ```go
-type IdentityLeaseSpec struct {
+type LeaseCandidateSpec struct {
+  // The fields BinaryVersion and CompatibilityVersion will be mandatory labels instead of fields in the spec
 	// BinaryVersion indicates the binary version of the component
-	BinaryVersion string
-
+	// BinaryVersion string
 	// Compatibility Version indicates the compatibility version of the component. Must be less than or equal to the binary version
-	CompatibilityVersion string
+	// CompatibilityVersion string
+
+	// CanLeadLease is in the format <namespace>/<name>, indicating the namespace and name of the lease that the candidate may lead
+	CanLeadLease string
```

Comment:
why namespace? Can't we make this local? Cannot imagine a usecase for cross-namespace setups. Also this might be a security problem or at least needs a discussion how to protect a lease from being hijacked.

Reply:
cc @jpbetz. I'm overall +1 on making it local though I'm not sure if we have any cases of cross-namespace setups.

Reply:
Still open.